less/_grid.less

```diff
@@ -134,7 +134,22 @@
 		max-width: 1280px;
 		margin: 0 auto;
 	}
+	// when there is no container for the grid remove the negative margins
+	&.no-container {
```

Comment:
This class, added to the parent `.grid`, makes the grid work without a container by removing the negative margin.